galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/ProvisionedAccountCreditTests.java

```diff
@@ -58,6 +56,11 @@ public class ProvisionedAccountCreditTests {
      * credit the account, then checks the balance again. The test passes if the
      * final balance is equal to the old balance + the credited amount.
      * 
+     * This test is an improved version of BasicAccountCreditTest.
+     * Log in is not hard coded and makes use of the cps.properties file to get the credentials.
+     * Navigating through menus is not hard coded and makes use of method such as '.getBalance()' so handle this
+     * which makes the test focus on the test objective itself.
+     * 
```

Comment:
There needs to be some information about the Simbank Manager - application manager that understands the application under test, and in this case provides XYZ